plugin.js

```diff
@@ -81,6 +81,7 @@ function fastifyCachingPlugin (instance, options, next) {
   instance.addHook('onRequest', etagHandleRequest)
   instance.addHook('onSend', etagOnSend)
 
+  instance[Symbol.for('fastify-caching.registered')] = true
```

Comment:
Thy this change?

Reply:
Primarily so it's easier to test. The module does a couple different types of caching and neither of them are as easy to test for proper registration as the cookie module is.